app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -130,6 +130,15 @@ def sync
       respond_for_async :resource => task
     end
 
+    api :POST, "/repositories/:id/export", N_("Export a repository")
+    param :id, :identifier, :required => true, :desc => N_("repository ID")
+    param :since, String, :desc => N_("ISO 8601 date of last export (ex: 2010-01-01T12:00:00), useful for exporting deltas.")
```

Comment:
I would like to use ISO8601 everywhere if possible. It looks like the first example is YYYY/MM/DD and the second is MM/DD/YYYY.

Ideally, these would all be ISO8601 (including API calls) and then converted to the user's preferred format during localization for display on the web UI.